api/controllers/UserController.js

```diff
@@ -104,8 +104,8 @@ module.exports = {
   update: function (req, res) {
     var attrs = _.pick(req.allParams(), [
       'name', 'bio', 'avatar_url', 'banner_url', 'twitter_name', 'linkedin_url', 'facebook_url',
-      'email', 'send_email_preference', 'work', 'intention', 'extra_info',
-      'new_notification_count', 'settings'
+      'email', 'send_email_preference', 'daily_digest', 'work', 'intention', 'extra_info',
```

Comment:
accidental regression here

if we'd been a little bit more coordinated, i would have suggested that you create a "settings" jsonb column on the user table -- because i just did that to set up daily/weekly/never for digest emails